charts/stable/rtorrent-rutorrent/values.yaml

```diff
@@ -51,19 +51,49 @@ service:
       rtinc:
         enabled: true
         port: 50000
+        
+  rthealth:
+    enabled: true
+    ports:
+      rthealth:
+        enabled: true
```

Comment:
No need to create services at all for probes. Unless those need to be accessible from a user, remove
Otherwise add to questions too